apps/slack-node/src/services/monday-service.js

```diff
@@ -0,0 +1,89 @@
+const initMondayClient = require('monday-sdk-js');
```

Comment:
Let's use the API team's SDK here instead ([docs](https://www.npmjs.com/package/@mondaydotcomorg/api)). It has better graphql support. Plus, we will deprecate server side usage of the `monday-sdk-js` package eventually. 

```suggestion
const { ApiClient } = require('@mondaydotcomorg/api');
const client = new ApiClient({ token });

[...]

const response = await client.request(query, variables)
const columnData = response.data.data.items[0].column_values[0];
```

Just double check the object shape of `response` before using this code. I can't remember if you get the data using `response.data.data.items...` or `response.data.items...`

Reply:
Done! and respone did need to update - data field is removed from all